technology/aws/src/main/java/com/github/mizool/technology/aws/dynamodb/Configuration.java

```diff
@@ -1,19 +1,36 @@
+/*
+ * Copyright 2017-2020 incub8 Software Labs GmbH
+ * Copyright 2017-2020 protel Hotelsoftware GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package com.github.mizool.technology.aws.dynamodb;
 
 import com.amazonaws.regions.Regions;
+import com.github.mizool.core.configuration.Config;
+import com.github.mizool.core.configuration.PropertyNode;
 
 public class Configuration
 {
-    private static final String DEFAULT_REGION = Regions.EU_CENTRAL_1.name();
+    private static final PropertyNode CONFIG = Config.systemProperties().child("aws");
 
     public Regions getAwsRegion()
     {
-        String region = System.getProperty("aws.region", DEFAULT_REGION);
-        return Regions.valueOf(region);
+        return CONFIG.child("region").convertedValue(Regions::valueOf).read().orElse(Regions.EU_CENTRAL_1);
```

Comment:
Extract `CONFIG.child("region").convertedValue(Regions::valueOf)` to a `REGION` constant.